Integrations/src/main/java/io/deephaven/integrations/python/PythonDeephavenSession.java

```diff
@@ -295,34 +273,48 @@ private Object maybeUnwrap(PyObject o) {
     }
 
     @Override
-    public Set<String> getVariableNames() {
-        return Collections.unmodifiableSet(scope.getKeys().collect(Collectors.toSet()));
+    protected Set<String> getVariableNames(Predicate<String> allowName) {
```

Comment:
`maybeUnwrap(Object)` a few lines above this is apparently unused and functionally the same as `unwrapObject`... should it be removed?
Likewise, I'm convinced that `maybeUnwrap(PyObject)` could just be replaced with `unwrapObject` calls.